resources/vue/src/components/account/orders/OrderListItem.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <tr>
+        <th scope="row">
+            <a :href="'/profile/order/' + order.id" class="list-group-item list-group-item-action">{{ order.id }}</a>
+        </th>
+        <td>{{ order.created_at }}</td>
+        <td>{{ order.price }}$</td>
+        <td>{{ order.paid ? 'yes' : 'no' }}</td>
+        <td>{{ order.trip.hotel.country }}</td>
+        <td>{{ order.trip.hotel.name}}</td>
```

Comment:
Very good use of object, but consider flattening the data, if that's possible. Here it's absolutely fine as it is now.